DiscordBot.Modules/CommandModules/PollModule.cs

```diff
@@ -110,5 +118,19 @@ public async Task EndPollAsync(string idOrUrl, [Remainder] string message = "")
                 //await err.DeleteAsync();
             }
         }
+
+        private bool CheckPollMsg(IUserMessage msg)
+        {
+            //Console.WriteLine($"`{msg.Embeds.First().Fields.Count()}Desc");
+            if ((msg.Content != "") ||
+                (msg.Embeds.Count != 1) ||
+                (msg.Embeds.First().Title == "") ||
```

Comment:
msg.Embeds.First() in eine eigene variable auslagern. First() ist zu inperformant, um es immer wieder aufzurufen.